src/Models/HydrostaticFreeSurfaceModels/update_hydrostatic_free_surface_model_state.jl

```diff
@@ -17,7 +17,7 @@ hydrostatic pressure) to the current model state.
 """
 update_state!(model::HydrostaticFreeSurfaceModel) = update_state!(model, model.grid)
 
-function update_state!(model::HydrostaticFreeSurfaceModel, grid)
+function update_state!(model::HydrostaticFreeSurfaceModel, callbacks=[])
```

Comment:
why this didn't become 

```Julia
update_state!(model::HydrostaticFreeSurfaceModel, grid, callbacks=[])
```

Reply:
I must have done this by mistake, looking now I don't think `grid` gets used anywhere in that function though?

Reply:
It’s used to differentiate a method for when the grid is single-column. See 

https://github.com/CliMA/Oceananigans.jl/blob/a86ef32581192d82a655e1b2ff4411e79917a379/src/Models/HydrostaticFreeSurfaceModels/single_column_model_mode.jl#L50

I believe I fixed it in #2786

Reply:
I believe we also need

```julia
update_state!(model::HydrostaticFreeSurfaceModel, callbacks=[]) = update_state!(model, model.grid, callbacks)
```